src/components/PromiseChart/index.js

```diff
@@ -8,7 +8,7 @@ import useStyles from "./useStyles";
 function PromiseChart({ chartLinks, title, description }) {
   const classes = useStyles();
 
-  if (!chartLinks.length) {
+  if (!(chartLinks && chartLinks.length)) {
```

Comment:
```suggestion
  if (!chartLinks?.length) {
```

[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) FTW!